patroni/dcs/consul.py

```diff
@@ -168,7 +168,10 @@ def create_session(self):
                 time.sleep(5)
 
     def set_ttl(self, ttl):
-        if self._client.http.set_ttl(ttl/2.0):  # Consul multiplies the TTL by 2x
+        ttl_ = ttl/2.0   # Consul multiplies the TTL by 2x
+        if ttl_ < 10.0:   # Minimal TTL for consul = 10, else gives "Error 500"
```

Comment:
There is a [documented](https://www.consul.io/docs/agent/options.html#session_ttl_min) way to make a TTL shorter than 10 seconds.
I think we should just omit a warning here instead of silently changing the value to 10.

Reply:
Reliable software must work with graceful degradation.  "Warnings" are not excuse.

May be better will be ask Consul Agent about self (/v1/agent/self) in Consul constructor and get "SessionTTLMin" in milliseconds. 

OK?

Reply:
Agreed, but in case if ttl is smaller than SessionTTLMin it should omit error message and bail out.
Also don't forget that if default value is not changed, you will probably get `"SessionTTLMin": 0` from `/v1/agent/self`.

Reply:
Oh, actually TTL could be changed dynamically and existing if it is too small is not a good idea, just printing a warning should be enough.